docs/public-api.asciidoc

```diff
@@ -205,6 +205,8 @@ WARNING: Avoid defining too many user-specified labels.
 Defining too many unique fields in an index is a condition that can lead to a
 {ref}/mapping.html#mapping-limit-settings[mapping explosion].
 
+Dots (`.`) in the keys are not allowed. In case you have a `.` in your label key, it will be replaced by `_`. For example `foo.bar` will be stored as `foo_bar`.
```

Comment:
Using `foo.bar` as an example of a label's key might be confusing as the following code sample has `foo` as the key and `bar` as the value.

Reply:
Changed to `a.b`.